app/views/ReportView/components/AnalystComments/index.tsx

```diff
@@ -108,7 +108,8 @@ const AnalystComments = ({
   const handleSign = useCallback(async (signed: boolean, updatedSignature: SignatureType) => {
     setIsSigned(signed);
     setSignatures(updatedSignature);
-  }, [setIsSigned]);
+    await queryClient.refetchQueries({ queryKey: ['report-signatures', report?.ident] });
+  }, [setIsSigned, queryClient]);
```

Comment:
missing dependency `report?.ident` , we don't need setSignatures since this refetch would force the previous `useEffect` to update the signatures

we probably don't need the `await/async` here as well, since all the hooks deal with things after API calls, and this function is called only after the API for the signature card is finished

incidentally, we should update the interaction where a user clicked on `sign` or `remove` on the signature card, and disable the button to press it when API call is happening -- so they don't spam the endpoint for fun